.github/CODEOWNERS

```diff
@@ -961,6 +961,9 @@
 # PRLabel: %Communication - Resource Manager
 /sdk/communication/Azure.ResourceManager.Communication/            @archerzz @ArcturusZhang @ArthurMa1978
 
+# ServiceLabel: %Container Orchestrator Runtime
+/sdk/containerorchestratorruntime/                                 @HE-Xinyu @ddadaal
```

Comment:
```suggestion
# PRLael: %Container Orchestrator Runtime
/sdk/containerorchestratorruntime/Azure.ResourceManager.*/         @HE-Xinyu @ddadaal

# ServiceLabel: %Container Orchestrator Runtime
# ServiceOwners:                                                   @HE-Xinyu @ddadaal
```

As written, this was not valid, did not follow the pattern for management libraries nor have the support entry needed.